_episodes_rmd/03-raster-reproject-in-r.Rmd

```diff
@@ -50,16 +50,6 @@ episode, we learned how to layer a raster file on top of a hillshade for a nice
 looking basemap. In this episode, all of our data were in the same CRS. What
 happens when things don't line up?
 
-We will use the `raster` and `rgdal` packages in this episode.
-
-```{r load-libraries }
```

Comment:
you can't remove this, you can hide it to avoid the repetition, but it's needed for the episode to render

Reply:
Added above